json-guides/microprofile-config.json

```diff
@@ -12,58 +12,129 @@
+                  "description": [
+                    "temp"
+                  ]
+                }
             ]
         },
         {
-            "name": "ConfigureViaInject",
-            "title": "Configuring with Default Injection Value",
+            "name": "ExampleScenario",
+            "title": "Example: xxx",
             "description": [
-                "Microprofile-Config provides ways to inject configured values into your beans using the <code>@Inject</code> and the <code>@ConfigProperty</code> qualifier"
+                "temp"
+            ]
+        },
+        {
+            "name": "AddLibertyMPFaultTolerance",
+            "title": "Enabling MicroProfile Config",
+            "description": [
+                "Begin by enabling the MicroProfile Config feature in your <code>server.xml</code> file."
             ],
             "instruction": [
-                "Add the <code>@Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")</code> on line 9, after the <code>private Config config</code> or click <action title='Configure inject using ConfigProperty' onclick=\"microprofileConfigCallBack.addConfigInjectButton(event)\"><b>@Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")</b></action>. Then, click <action title='Save' onclick=\"microprofileConfigCallBack.saveButton(event)\"><b>Save</b></action> on the editor menu pane."
+              "Add <b>&lt;feature>mpConfig-1.1&lt;/feature>."
+            ]
+        },
+        {
+            "name": "ConfigureViaInject",
+            "title": "Injecting config properties through annotations",
+            "description": [
+                "After you modify your <code>server.xml</code> file to <a>include the mpConfig feature</a>, a single configuration property value can be injected directly into your code using the <code>@Inject</code> and the <code>@ConfigProperty</code> annotations. The injected value is set when the application starts and is static.<br>", 
+                "Injecting a configuration value this way is recommended for mandatory properties. If no configured value exists for this property, a <code>DeploymentException</code> will be thrown during startup."
             ],
+            "instruction": [
+                "To inject a <code>download-url</code> configuration property into the code, add the following <code>@Inject</code> and <code>@ConfigProperty</code> annotations to line 6 before declaring the <code>String download-url</code>, or click  <action title='Inject a ConfigProperty'><b>@Inject @ConfigProperty(name=\"download-url\")</b></action>. Then, click <action title='Save'><b>Save</b></action> on the editor menu pane.<br><br><note>Note: You have not yet specified a value for the <code>download-url</code> configuration property.  Execution of this code at this time would cause a <code>DeploymentException</code> to be thrown since a value is required.</note>"
```

Comment:
There is no callback to inject the line of code into the editor. Please open an issue to complete this step. We also need to decide whether to use a browser to get the DeploymentException or not.

Reply:
Issue #53 opened.